Segment/Classes/SEGAnalyticsConfiguration.m

```diff
@@ -42,6 +42,7 @@ @interface SEGAnalyticsConfiguration ()
 @property (nonatomic, copy, readwrite) NSString *writeKey;
 @property (nonatomic, strong, readonly) NSMutableArray *factories;
 @property (nonatomic, strong) SEGAnalyticsExperimental *experimental;
+@property (nonatomic, strong) NSString *instanceId;
```

Comment:
should we make `instanceId` readonly?

Reply:
no, it's in a private interface already and needs to be set.